framework-cloud/framework-cloud-api/src/main/java/org/kie/cloud/api/deployment/CommandExecutionResult.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+package org.kie.cloud.api.deployment;
+
+import java.io.ByteArrayOutputStream;
+
+public class CommandExecutionResult {
+    private ByteArrayOutputStream output;
```

Comment:
Who's going to close those streams and when?

Reply:
It is not necessary to close these streams because ByteArrayOutputStream does not consume any resources. It consumes only memory. Even close method is empty for these streams. It is also problem to find right place where to close them in this case.